lib/data_migrate/database_tasks.rb

```diff
@@ -22,9 +22,9 @@ def with_temporary_connection_for_each(env: ActiveRecord::Tasks::DatabaseTasks.e
         end
       end
 
-      def with_temporary_connection(db_config) # :nodoc:
-        with_temporary_pool(db_config) do |pool|
-          yield pool.connection
+      def with_temporary_connection(db_config, clobber: false, &block) # :nodoc:
```

Comment:
changing the method definition would cause issues for older rails versions (tested against 7.0, 7.1 already supports the clobber keyword)

Would you be able to patch that for rails 7.0?